resources/kprove.mak

```diff
@@ -79,7 +79,7 @@ $(K_REPO_DIR):
 	cd $(K_REPO_DIR) \
 		&& git reset --hard $(K_VERSION) \
 		&& git submodule update --init --recursive \
```

Comment:
is the above `git submodule update` still needed with this change?

Reply:
That line is just an experiment from Everett. I'll delete it.